lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,17 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_code, from: nil, till: nil, outcome: nil, sbg_key: nil, primary: false)
```

Comment:
Sbg_key defaulten naar de sbg_key van de lijst is nog wel een goeie. Outcome is volgens mij niet verplicht.